spec/models/claim/pii_scrubber_spec.rb

```diff
@@ -0,0 +1,97 @@
+require "rails_helper"
+
+RSpec.describe Claim::PiiScrubber, type: :model do
+  let(:over_two_months_ago) { 2.months.ago - 1.day }
+
+  it "does not delete details from a submitted claim" do
+    freeze_time
+    claim = create(:claim, :submitted, updated_at: over_two_months_ago)
+
+    Claim::PiiScrubber.new.scrub_completed_claims
+    cleaned_claim = Claim.find(claim.id)
+
+    expect(cleaned_claim.attributes).to eq(claim.attributes)
+  end
+
+  it "does not delete details from an approved but unpaid claim" do
+    freeze_time
```

Comment:
Might be worth wrapping `freeze_time` in a block here, just to make sure the time unfreezes at the end of the spec (https://api.rubyonrails.org/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-freeze_time)